charts/images.yaml

```diff
@@ -100,7 +100,21 @@ images:
   sourceRepository: github.com/gardener/cloud-provider-gcp
```

Comment:
Please incorporate all the gcp CCM versions released in https://github.com/gardener/gardener/issues/7783.